examples/with-passport-and-next-connect/lib/session.js

```diff
@@ -29,13 +29,13 @@ export default function session({ name, secret, cookie: cookieOpts }) {
 
     // We are proxying res.end to commit the session cookie
     const oldEnd = res.end
-    res.end = async function resEndProxy(...args) {
+    res.end = function resEndProxy(...args) {
       if (res.finished || res.writableEnded || res.headersSent) return
       if (cookieOpts.maxAge) {
         req.session.maxAge = cookieOpts.maxAge
       }
 
-      const token = await createLoginSession(req.session, secret)
+      const token = createLoginSession(req.session, secret)
```

Comment:
This looks like a bug. 

The await is necessary since the function is async as seen here:

https://github.com/vrun1208/next.js/blob/b06c83470303c03f64f3be7eb9c2bd604a41e394/examples/with-passport-and-next-connect/lib/auth.js#L3